static/app/views/performance/transactionSummary/transactionOverview/index.tsx

```diff
@@ -149,6 +149,22 @@ function OverviewContentWrapper(props: ChildProps) {
     error: totalCountQueryError,
   } = totalCountQueryData;
 
+  const isTableEmpty = !isLoading && !error && tableData.data.length === 0;
+
+  useEffect(() => {
+    if (mepContext.isMetricsData && organization.isDynamicallySampled) {
+      trackAnalytics('dynamic_sampling_transaction_summary.baseline', {
+        organization,
```

Comment:
@matejminar is `organization` enough or do we want to gather other info?

Reply:
We get the URL for free, which should be enough (although query params might be missing to precisely locate the gap). Let's merge and see.